base.sh

```diff
@@ -28,6 +28,13 @@ function runnerArch() {
   cat "${JOB}/${CUSTOM_ENV_RUNNER}/config.json" | jq -r '.runnerArch'
 }
 
+function waitForUserLogout() {
+	while (( $(who -s | wc -l)  > 0 )); do
```

Comment:
This runs on the scheduling machine, not on the test runner. Need to ssh into it. Note that the `ssh` session created to do the check might interfere with the check itself.

Reply:
Sorry I totally missed your review, I updated it with ssh. When testing it manually it seems that running that command over ssh does not count as login because I received `0` when nobody was logged in. Also this might make the runners look stuck I'm not sure, but we can't print to the job log during cleanup anymore.